### ISSUE TITLE
[Bug]: gpt-3.5-turbo batch_completion end point

### ISSUE THREAD

**Issue by jyzhang111** [BerriAI/litellm#549]
### What happened?

I ran the code on the front page for gpt-3.5-turbo, and ran into the bug as shown:

import litellm
import os
from litellm import batch_completion

os.environ['OPENAI_API_KEY'] = ""

responses = batch_completion(
    model="gpt-3.5-turbo",
    messages = [
        [
            {
                "role": "user",
                "content": "good morning? "
            }
        ],
        [
            {
                "role": "user",
                "content": "what's the time? "
            }
        ]
    ]
)

### Relevant log output

```shell
File /Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/site-packages/litellm/main.py:1294, in batch_completion(model, messages, functions, function_call, temperature, top_p, n, stream, stop, max_tokens, presence_penalty, frequency_penalty, logit_bias, user, return_async, api_key, api_version, api_base, force_timeout, top_k, custom_llm_provider)
   1291                 completions.append(future)
   1293     # Retrieve the results from the futures
-> 1294     results = [future.result() for future in completions]
   1295 return results

File /Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/site-packages/litellm/main.py:1294, in <listcomp>(.0)
   1291                 completions.append(future)
   1293     # Retrieve the results from the futures
-> 1294     results = [future.result() for future in completions]
   1295 return results

File /Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/concurrent/futures/_base.py:449, in Future.result(self, timeout)
    447     raise CancelledError()
    448 elif self._state == FINISHED:
--> 449     return self.__get_result()
    451 self._condition.wait(timeout)
    453 if self._state in [CANCELLED, CANCELLED_AND_NOTIFIED]:

File /Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/concurrent/futures/_base.py:401, in Future.__get_result(self)
    399 if self._exception:
    400     try:
--> 401         raise self._exception
    402     finally:
    403         # Break a reference cycle with the exception in self._exception
    404         self = None

File /Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/concurrent/futures/thread.py:58, in _WorkItem.run(self)
     55     return
     57 try:
---> 58     result = self.fn(*self.args, **self.kwargs)
     59 except BaseException as exc:
     60     self.future.set_exception(exc)

File /Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/site-packages/litellm/utils.py:718, in client.<locals>.wrapper(*args, **kwargs)
    714     if (
    715         liteDebuggerClient and liteDebuggerClient.dashboard_url != None
    716     ):  # make it easy to get to the debugger logs if you've initialized it
    717         e.message += f"\n Check the log in your dashboard - {liteDebuggerClient.dashboard_url}"
--> 718 raise e

File /Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/site-packages/litellm/utils.py:677, in client.<locals>.wrapper(*args, **kwargs)
    674             return cached_result
    676 # MODEL CALL
--> 677 result = original_function(*args, **kwargs)
    678 end_time = datetime.datetime.now()
    679 if "stream" in kwargs and kwargs["stream"] == True:
    680     # TODO: Add to cache for streaming

File /Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/site-packages/litellm/timeout.py:53, in timeout.<locals>.decorator.<locals>.wrapper(*args, **kwargs)
     51     local_timeout_duration = kwargs["request_timeout"]
     52 try:
---> 53     result = future.result(timeout=local_timeout_duration)
     54 except futures.TimeoutError:
     55     thread.stop_loop()

File /Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/concurrent/futures/_base.py:456, in Future.result(self, timeout)
    454     raise CancelledError()
    455 elif self._state == FINISHED:
--> 456     return self.__get_result()
    457 else:
    458     raise TimeoutError()

File /Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/concurrent/futures/_base.py:401, in Future.__get_result(self)
    399 if self._exception:
    400     try:
--> 401         raise self._exception
    402     finally:
    403         # Break a reference cycle with the exception in self._exception
    404         self = None

File /Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/site-packages/litellm/timeout.py:42, in timeout.<locals>.decorator.<locals>.wrapper.<locals>.async_func()
     41 async def async_func():
---> 42     return func(*args, **kwargs)

File /Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/site-packages/litellm/main.py:1211, in completion(model, messages, functions, function_call, temperature, top_p, n, stream, stop, max_tokens, presence_penalty, frequency_penalty, logit_bias, user, deployment_id, **kwargs)
   1208     return response
   1209 except Exception as e:
   1210     ## Map to OpenAI Exception
-> 1211     raise exception_type(
   1212         model=model, custom_llm_provider=custom_llm_provider, original_exception=e, completion_kwargs=args,
   1213     )

File /Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/site-packages/litellm/utils.py:2846, in exception_type(model, original_exception, custom_llm_provider, completion_kwargs)
   2844 # don't let an error with mapping interrupt the user from receiving an error from the llm api calls
   2845 if exception_mapping_worked:
-> 2846     raise e
   2847 else:
   2848     raise original_exception

File /Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/site-packages/litellm/utils.py:2265, in exception_type(model, original_exception, custom_llm_provider, completion_kwargs)
   2259     if "This model's maximum context length is" in original_exception._message:
   2260         raise ContextWindowExceededError(
   2261             message=str(original_exception),
   2262             model=model,
   2263             llm_provider=original_exception.llm_provider
   2264         )
-> 2265     raise original_exception
   2266 elif model:
   2267     error_str = str(original_exception)

File /Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/site-packages/litellm/main.py:391, in completion(model, messages, functions, function_call, temperature, top_p, n, stream, stop, max_tokens, presence_penalty, frequency_penalty, logit_bias, user, deployment_id, **kwargs)
    383 except Exception as e:
    384     ## LOGGING - log the original exception returned
    385     logging.post_call(
    386         input=messages,
    387         api_key=api_key,
    388         original_response=str(e),
    389         additional_args={"headers": litellm.headers},
    390     )
--> 391     raise e
    393 if "stream" in optional_params and optional_params["stream"] == True:
    394     response = CustomStreamWrapper(response, model, custom_llm_provider="openai", logging_obj=logging)

File /Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/site-packages/litellm/main.py:373, in completion(model, messages, functions, function_call, temperature, top_p, n, stream, stop, max_tokens, presence_penalty, frequency_penalty, logit_bias, user, deployment_id, **kwargs)
    371 ## COMPLETION CALL
    372 try:
--> 373     response = openai.ChatCompletion.create(
    374         model=model,
    375         messages=messages,
    376         headers=litellm.headers, # None by default
    377         api_base=api_base, # thread safe setting base, key, api_version
    378         api_key=api_key,
    379         api_type="openai",
    380         api_version=api_version, # default None
    381         **optional_params,
    382     )
    383 except Exception as e:
    384     ## LOGGING - log the original exception returned
    385     logging.post_call(
    386         input=messages,
    387         api_key=api_key,
    388         original_response=str(e),
    389         additional_args={"headers": litellm.headers},
    390     )

File /Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/site-packages/openai/api_resources/chat_completion.py:25, in ChatCompletion.create(cls, *args, **kwargs)
     23 while True:
     24     try:
---> 25         return super().create(*args, **kwargs)
     26     except TryAgain as e:
     27         if timeout is not None and time.time() > start + timeout:

File /Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/site-packages/openai/api_resources/abstract/engine_api_resource.py:153, in EngineAPIResource.create(cls, api_key, api_base, api_type, request_id, api_version, organization, **params)
    127 @classmethod
    128 def create(
    129     cls,
   (...)
    136     **params,
    137 ):
    138     (
    139         deployment_id,
    140         engine,
   (...)
    150         api_key, api_base, api_type, api_version, organization, **params
    151     )
--> 153     response, _, api_key = requestor.request(
    154         "post",
    155         url,
    156         params=params,
    157         headers=headers,
    158         stream=stream,
    159         request_id=request_id,
    160         request_timeout=request_timeout,
    161     )
    163     if stream:
    164         # must be an iterator
    165         assert not isinstance(response, OpenAIResponse)

File /Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/site-packages/openai/api_requestor.py:298, in APIRequestor.request(self, method, url, params, headers, files, stream, request_id, request_timeout)
    277 def request(
    278     self,
    279     method,
   (...)
    286     request_timeout: Optional[Union[float, Tuple[float, float]]] = None,
    287 ) -> Tuple[Union[OpenAIResponse, Iterator[OpenAIResponse]], bool, str]:
    288     result = self.request_raw(
    289         method.lower(),
    290         url,
   (...)
    296         request_timeout=request_timeout,
    297     )
--> 298     resp, got_stream = self._interpret_response(result, stream)
    299     return resp, got_stream, self.api_key

File /Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/site-packages/openai/api_requestor.py:700, in APIRequestor._interpret_response(self, result, stream)
    692     return (
    693         self._interpret_response_line(
    694             line, result.status_code, result.headers, stream=True
    695         )
    696         for line in parse_stream(result.iter_lines())
    697     ), True
    698 else:
    699     return (
--> 700         self._interpret_response_line(
    701             result.content.decode("utf-8"),
    702             result.status_code,
    703             result.headers,
    704             stream=False,
    705         ),
    706         False,
    707     )

File /Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/site-packages/openai/api_requestor.py:765, in APIRequestor._interpret_response_line(self, rbody, rcode, rheaders, stream)
    763 stream_error = stream and "error" in resp.data
    764 if stream_error or not 200 <= rcode < 300:
--> 765     raise self.handle_error_response(
    766         rbody, rcode, resp.data, rheaders, stream_error=stream_error
    767     )
    768 return resp

InvalidRequestError: We could not parse the JSON body of your request. (HINT: This likely means you aren't using your HTTP library correctly. The OpenAI API expects a JSON payload, but what was sent was not valid JSON. If you have trouble figuring out how to fix this, please contact us through our help center at help.openai.com.)
```

**Comment by ghost** [comment]
Hi @jyzhang111 - acknowledging the issue. @ishaan-jaff assigning to you, as you'd worked on this. Can take a look, if you're busy atm.

**Comment by ishaan-jaff** [comment]
@jyzhang111 this is fixed: https://github.com/BerriAI/litellm/commit/4ae8a71aa3eed0ad53c15efe9314042bcdf24759

**Comment by ishaan-jaff** [comment]
thanks for raising and apologies for the error

**Comment by ishaan-jaff** [comment]
will let you know once it's redeployed

**Comment by ishaan-jaff** [comment]
fixed on litellm 0.3.0 and higher
